docs/components/phpcs.md

```diff
@@ -7,7 +7,7 @@ During the installation of the testing suite a file called `phpcs.xml` is added
 the root of the repository which refers to the coding standard. To make
 adjustments to the coding standard this file can be edited and committed.
 
-[PHPCS](https://github.com/FriendsOfPHP/PHP-CS-Fixer)
+[PHPCS](https://github.com/FriendsOfPHP/PHP-CS-Fixer) \
```

Comment:
Typo?

Reply:
\ is a newline so a blank line is not needed. I use it a lot and it's probably accidental that everything else doesn't need it because the changes are mostly sums or entire paragraphs without needing explicit newlines.